src/queueserver.ts

```diff
@@ -450,3 +450,33 @@ export const modifyQueue = async(op: QueueOps): Promise<IPlanModify> => {
     console.log(res);
     return res.data;
 }
+
+export const incrementPosition = async(item1_uid: string, item2_uid: string): Promise<IPlanModify> => {
+    const res = await axiosInstance.post(`/queue/item/move`,
+        {
+            uid: item1_uid,
+            after_uid: item2_uid
+
+        });
+    console.log(res);
+    return res.data;
+}
+
+export const decrementPosition = async(item1_uid: string, item2_uid: string): Promise<IPlanModify> => {
+    const res = await axiosInstance.post(`/queue/item/move`,
+        {
+            uid: item1_uid,
+            before_uid: item2_uid
+        });
+    console.log(res);
+    return res.data;
+}
+
+export const addQueueStop = async(): Promise<IPlanModify> => {
+    const res = await axiosInstance.post(`/queue/item/add`,
```

Comment:
As mentioned in the previous PR, backticks may not be needed here.

Reply:
updated